test/common_utils.py

```diff
@@ -829,6 +829,10 @@ def make_video(size=DEFAULT_SIZE, *, num_frames=3, batch_dims=(), **kwargs):
     return datapoints.Video(make_image(size, batch_dims=(*batch_dims, num_frames), **kwargs))
 
 
+def make_video_tensor(*args, **kwargs):
+    return make_video(*args, **kwargs).as_subclass(torch.Tensor)
```

Comment:
Just to make sure: we need that to test the "shape getters" which may be called on pure tensors within a video-specific kernel. But otherwise, we don't consider pure tensors to be videos. Right?

Reply:
Not quite. In 99% of the cases plain tensors are images, but there are some exceptions:

1. https://github.com/pytorch/vision/blob/16d62e3072955bd92b76a4ae73fefa73ecc9ee3e/torchvision/transforms/v2/functional/_meta.py#L228
2. https://github.com/pytorch/vision/blob/16d62e3072955bd92b76a4ae73fefa73ecc9ee3e/torchvision/transforms/v2/functional/_meta.py#L279
3. https://github.com/pytorch/vision/blob/16d62e3072955bd92b76a4ae73fefa73ecc9ee3e/torchvision/transforms/v2/functional/_temporal.py#L17

Per name, 1. and 2. treat plain tensors as bounding boxes. 3. treats plain tensors as video.